packages/story-editor/src/components/library/panes/text/textSets/textSet.js

```diff
@@ -113,7 +113,9 @@ function TextSet({ id, elements, translateY, translateX, ...rest }, ref) {
     [onClick]
   );
 
-  const renderImages = WEB_STORIES_DISABLE_OPTIMIZED_RENDERING !== 'true';
+  const renderImages =
+    typeof WEB_STORIES_DISABLE_OPTIMIZED_RENDERING !== 'undefined' &&
```

Comment:
Since the default should be `false`, shouldn't it be ok here to check for `WEB_STORIES_DISABLE_OPTIMIZED_RENDERING !== 'true';` only since it also behaves correctly if not set?

Reply:
Or changing the constant name (and usage):
`const omitRenderingImages = typeof WEB_STORIES_DISABLE_OPTIMIZED_RENDERING !== 'undefined' && WEB_STORIES_DISABLE_OPTIMIZED_RENDERING === 'true';`

Or
`const renderImages = typeof WEB_STORIES_DISABLE_OPTIMIZED_RENDERING === 'undefined' || WEB_STORIES_DISABLE_OPTIMIZED_RENDERING !== 'true';`

That would perhaps fix the test failures as well since these seem to be all related to the text panel.